pdg/src/builder.rs

```diff
@@ -44,6 +44,7 @@ impl EventKindExt for EventKind {
         use EventKind::*;
         Some(match *self {
             CopyPtr(lhs) => lhs,
+            // Question: this takes a pointer as input and produces another?
             Field(ptr, ..) => ptr,
```

Comment:
> Question: this takes a pointer as input and produces another?

That's what I would expect.  In the static analysis, we have a similar notion of `PlaceElem::Field` as a "pointer projection" that takes a pointer to the struct and produces a pointer to the field.  This is similar to (some uses of) `offset`, which takes a pointer to (the first element of) an array and produces a pointer to an element, and also similar to [LLVM's GetElementPtr operation](https://llvm.org/docs/GetElementPtr.html).